_content/essays/amazing-transformations-theri-uppalavanna_tathaloka.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Amazing Transformations of Arahant Theri Uppalavanna"
+authors:
+  - tathaloka
+external_url: "https://www.bhikkhuni.net/wp-content/uploads/2013/08/Transformations-of-Arahant-Theri-Uppalavanna1.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1guoCP8kjUXw2iikDatB0_RAaiCI6N5N_/view?usp=sharing"
+course: medieval
+tags:
+  - characters
+  - imagery
```

Comment:
`imagery` isn't quite right here...  But let's add `nuns` for sure as her changing story is a reflection of changing attitudes towards them.